Unix/protocol/protocol.c

```diff
@@ -8,6 +8,9 @@
 */
 
 #include <assert.h>
+#include <sys/types.h>
```

Comment:
I don't fully understand this code ... @yakman2020 can you please review this too? I'd like to see Bruce review this prior to checkin please.